browser/ui/views/frame/brave_browser_view.cc

```diff
@@ -404,6 +405,12 @@ speedreader::SpeedreaderBubbleView* BraveBrowserView::ShowSpeedreaderBubble(
 #endif
 }
 
+bool BraveBrowserView::HasSelectedURL() const {
+  return static_cast<BraveOmniboxViewViews*>(
+             GetLocationBarView()->omnibox_view())
+      ->SelectedTextIsURL();
```

Comment:
I'm wondering if we should support `Copy clean link` when users selects URLs on web contents too. If so, I guess we might want to access ui::TextInputClient to check selected text.

Reply:
@sangwoo108 Could you post a screenshot of the menu you're talking about to make sure I understand your suggestion correctly?

Reply:
Hi, @fmarier . It is my understanding that this PR is trying to insert "Copy clean link" into app menu on mac when users select URL in omnibox. Please correct me if it's not.

So I was wondering if we should insert the same menu when users select URL on the web page.

<img width="605" alt="Screen Shot 2023-01-04 at 3 37 25 PM" src="https://user-images.githubusercontent.com/5474642/210498352-760289a6-e4c4-42f2-b1e1-7652240d46b5.png">

Reply:
Ah, interesting. That's a good idea. Not just on Mac, but on all desktop platforms too.

@sangwoo108 Would you mind filing a follow-up issue for this? You can give it the [`copy-clean-link`](https://github.com/brave/brave-browser/labels/copy-clean-link) label.

Reply:
Sorry for delayed response. Sure! Here it is. https://github.com/brave/brave-browser/issues/27676